index.js

```diff
@@ -18,4 +19,6 @@ module.exports = function (file, args, options) {
 			resolve({stdout: stdout, stderr: stderr});
 		});
 	});
+	promise.child = child;
```

Comment:
It should actually be the other way around. We should attach the `then` and `catch` methods of the promise to the child and return the child. This would make it much easier to use.